api/lib/infrastructure/monitoring-tools.js

```diff
@@ -71,10 +79,15 @@ function extractUserIdFromRequest(request) {
   return userId || '-';
 }
 
+function isStoreContainsApiRequest() {
+  return (asyncLocalStorage.getStore());
```

Comment:
Question : Peut-on être plus explicite en vérifiant que `getStore()` ne retourne pas `undefined` ?

Reply:
J'ai rajouté le undefined. 

Vu ensemble dans la doc, peut retourner undefined si jamais elle est appelé en dehors du contexte initialisé par asyncLocalStorage.run() et dans notre cas au niveau de la requête Hapi.

`Returns the current store. If called outside of an asynchronous context initialized by calling asyncLocalStorage.run() or asyncLocalStorage.enterWith(), it returns undefined.`